.github/workflows/test.yaml

```diff
@@ -88,22 +88,22 @@ jobs:
         include:
           - feature_version: 6.0.0 # You must ensure feature_version and tidb_version is matching!
             tidb_version: nightly
-            without_ngm: false
+            without_ngm: true
           - feature_version: 6.0.0
             tidb_version: nightly
             without_ngm: true
```

Comment:
These 2 are duplicated now, recommend to comment it instead of modifying `without_ngm`.